jbi/actions/default.py

```diff
@@ -95,15 +97,30 @@ def __init__(self, steps: dict[Operation, list[Callable]], **parameters):
 
     def __call__(self, context: ActionContext) -> ActionResult:
         """Called from `runner` when the action is used."""
+        has_produced_request = False
+
         for step in self.steps[context.operation]:
             context = context.update(current_step=step.__name__)
             try:
                 context = step(context=context, **self.parameters)
             except IncompleteStepError as exc:
                 # Step did not execute all its operations.
                 context = exc.context
+                statsd.incr(
+                    f"jbi.action.{context.jira.project.lower()}.incomplete.count"
```

Comment:
Can we use the whiteboard tag of the workflow instead of the jira project, since the whiteboard tag is more what we consider the "name" of the workflow? Same below in the `aborted` case